core/commands/root.go

```diff
@@ -132,14 +140,17 @@ var rootROSubcommands = map[string]*cmds.Command{
 			"stat":  objectStatCmd,
 		},
 	},
-	"refs": RefsCmd,
+	"refs": RefsROCmd,
 	//"resolve": ResolveCmd,
 }
 
-var RootRO = &cmds.Command{}
-
 func init() {
 	*RootRO = *Root
+
+	// sanitize readonly refs command
```

Comment:
i think it is useful, and also that safety depends on context. keep it disabled until we find a strong reason to enable it?